packages/react-core/src/components/Form/examples/FormFieldGroups.tsx

```diff
@@ -0,0 +1,388 @@
+import React from 'react';
+import {
+  Form,
+  FormGroup,
+  FormFieldGroup,
+  FormFieldGroupExpandable,
+  FormFieldGroupHeader,
+  TextInput,
+  Button
+} from '@patternfly/react-core';
+import TrashIcon from '@patternfly/react-icons/dist/esm/icons/trash-icon';
+
+export const FieldGroups: React.FunctionComponent = () => {
+  const initialValues = {
+    'form-expandable-field-groups-label1': '',
+    'form-expandable-field-groups-label2': '',
+    'form-expandable-field-groups-field-group1-label1': '',
+    'form-expandable-field-groups-field-group1-label2': '',
+    'form-expandable-field-groups-field-group2-label1': '',
+    'form-expandable-field-groups-field-group2-label2': '',
+    'form-expandable-field-groups-field-group3-label1': '',
+    'form-expandable-field-groups-field-group3-label2': '',
+    'form-expandable-field-groupsform-expandable-field-groups-field-group1-label1': '',
```

Comment:
I think `form-expandable-field-groupsform-expandable-field-groups-field-group1-label1` is referring to one of the nested fields? 

We could call it `field-group1-nested-group1-label1` - would clear things up a bit 🙂